packages/moonstone/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The following is a curated list of changes in the Enact moonstone module, newest
 ### Fixed
 
 - `moonstone/Scroller`, `moonstone/VirtualList.VirtualGridList`, and `moonstone/VirtualList.VirtualList` to remove event listeners properly
+- `moonstone/ContextualPopupDecorator` timing to read out `activator` button when the `ContextualPopup` is closed by pressing a back button of remote control
```

Comment:
No need to wrap 'activator' with backticks. It's just a name for a variable. I prefer 'back key' than 'back button'.